helm/holmes/values.yaml

```diff
@@ -42,17 +42,7 @@ serviceAccount:
   imagePullSecrets: []
   annotations: {}
 
-toolsets:
-  kubernetes/core:
-    enabled: true
-  kubernetes/logs:
-    enabled: true
-  robusta:
```

Comment:
I saw that robusta indeed enabled by default if dal was able to init.
Im not sure that this is the case with prometheus & internet toolset. am i missing something?